drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/migration/initial_mci_unique_id_creation_job.rb

```diff
@@ -6,6 +6,7 @@
 
 module HmisExternalApis::AcHmis::Migration
   class InitialMciUniqueIdCreationJob < BaseJob
+    queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
```

Comment:
All these jobs in `migration/` are not run anymore, fyi

Reply:
figured, though you never know 🤞